README.md

```diff
@@ -1,20 +1,22 @@
-[![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/195/9.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-l10n-columbia-201)
-[![Build Status](https://travis-ci.org/OCA/l10n-columbia.svg?branch=9.0)](https://travis-ci.org/OCA/l10n-columbia)
-[![Coverage Status](https://coveralls.io/repos/OCA/l10n-columbia/badge.svg?branch=9.0&service=github)](https://coveralls.io/github/OCA/l10n-columbia?branch=9.0)
-[![Code Climate](https://codeclimate.com/github/OCA/l10n-columbia/badges/gpa.svg)](https://codeclimate.com/github/OCA/l10n-columbia)
+# Colombian Localization for Odoo
 
-# Columbia localisation
-
-write me
 
 [//]: # (addons)
+Addons
+---------------
+addon | version | summary
+--- | --- | ---
+[l10n_co_res_partner](l10n_co_res_partner/) | 9.0.1.0.0 | Colombia - Extended Partner / Contact Module (Terceros)
+
 [//]: # (end addons)
 
-Translation Status
-------------------
-[![Transifex Status](https://www.transifex.com/projects/p/OCA-l10n-columbia-9-0/chart/image_png)](https://www.transifex.com/projects/p/OCA-l10n-columbia-9-0)
 
-----
+Feedback
+-----------------
+We want to improve constantly our software, therefore we love feedback. Feel free to comment at any time our code. 
+If you think something is missing please don't hesitate to drop a message.
+
+-----------------
```

Comment:
point to OCA documentation on this comment copy some other stable README for more information.